.ibm/images/Dockerfile

```
@@ -100,4 +100,18 @@ RUN go install github.com/kadel/pr-commenter@latest && \
     ibmcloud plugin install -f cloud-object-storage && \
     ibmcloud plugin install -f kubernetes-service
 
+# Install skopeo
+RUN apt-get update -y && \
+    apt-get install -y skopeo
+
+# Install umoci
+RUN curl -LO "https://github.com/opencontainers/umoci/releases/download/v0.4.7/umoci.amd64" && \ 
+    chmod +x umoci.amd64 && \
+    mv umoci.amd64 /usr/local/bin/umoci
+
+# Install opm
+RUN curl -LO "https://github.com/operator-framework/operator-registry/releases/download/v1.19.5/linux-amd64-opm" && \ 
```

```suggestion
RUN curl -LO "https://github.com/operator-framework/operator-registry/releases/download/v1.23.0/linux-amd64-opm" && \ 
```

Otherwise, the `opm generate dockerfile` command used in the install script is not recognized. This also aligns with the version of `opm` we are using in the operator repo: https://github.com/redhat-developer/rhdh-operator/blob/main/Makefile#L468